File: samples/c/tests/sample-ut-calculate.c
// UT| gcc -o test.out {src_file}
// UT| ./test.out


// UT[ ../sample.c * calculate
int calculate(int value) {
    return value * value;
}
// UT]


#include <stdio.h>

int main() {
    printf("calculate = %d\n", calculate(3));
    return 0;
}


// UT> calculate = 9

File: samples/go/tests/sample-ut-calculate.go
// UT| go run {src_file}

package main

import (
	"fmt"
)

// UT[ ../sample.go * calculate
func calculate(value int) int {
	return value * value
}

// UT]

func main() {
	fmt.Printf("calculate = %d\n", calculate(4))
}

// UT> calculate = 16
